config/services.xml

```diff
@@ -27,16 +26,9 @@
         <service id="Magento\MagentoCloud\DB\Data\Connection" autowire="false" />
         <service id="Magento\MagentoCloud\DB\Data\RelationshipConnection" autowire="false" />
         <service id="Magento\MagentoCloud\DB\PDOException" autowire="false" />
-        <service id="Magento\MagentoCloud\Filesystem\FileSystemException" autowire="false" />
+        <service id="Magento\MagentoCloud\Filesystem\FilesystemException" autowire="false" />
```

Comment:
What's wrong with the previous name? For me, such changes have no sense.
https://www.google.com/search?q=FileSystemException

Reply:
Wikipedia and other vendors have it like this https://en.wikipedia.org/wiki/File_system

Symfony: https://github.com/symfony/filesystem/blob/master/Filesystem.php
Laravel:  https://github.com/illuminate/filesystem

Reply:
I agree that this can be in both variants, but I don't see any value in changing `System` from uppercase to lowercase. For me `FileSystemException` looks better.